state-chain/runtime/src/lib.rs

```diff
@@ -1870,6 +1876,166 @@ impl_runtime_apis! {
+				arbitrum: pallet_cf_broadcast::PendingBroadcasts::<Runtime, ArbitrumInstance>::decode_non_dedup_len().unwrap_or(0) as u32,
+			}
+		}
+		fn cf_pending_tss_ceremonies() -> PendingTssCeremonies {
+			PendingTssCeremonies {
+				evm: pallet_cf_threshold_signature::PendingCeremonies::<Runtime, EvmInstance>::iter().collect::<Vec<_>>().len() as u32,
+				bitcoin: pallet_cf_threshold_signature::PendingCeremonies::<Runtime, BitcoinInstance>::iter().collect::<Vec<_>>().len() as u32,
+				polkadot: pallet_cf_threshold_signature::PendingCeremonies::<Runtime, PolkadotInstance>::iter().collect::<Vec<_>>().len() as u32,
+			}
+		}
+		fn cf_pending_swaps() -> u32 {
+			let swaps: Vec<_> = pallet_cf_swapping::SwapQueue::<Runtime>::iter().collect();
+			swaps.iter().fold(0u32, |acc, elem| acc + elem.1.len() as u32)
+		}
+		fn cf_open_deposit_channels() -> OpenDepositChannels {
+			let eth_channels: u32 = pallet_cf_ingress_egress::DepositChannelLookup::<Runtime, EthereumInstance>::iter().filter(|(_key, elem)| elem.expires_at > pallet_cf_chain_tracking::CurrentChainState::<Runtime, EthereumInstance>::get().unwrap().block_height).collect::<Vec<_>>().len() as u32;
+			let btc_channels: u32 = pallet_cf_ingress_egress::DepositChannelLookup::<Runtime, BitcoinInstance>::iter().filter(|(_key, elem)| elem.expires_at > pallet_cf_chain_tracking::CurrentChainState::<Runtime, BitcoinInstance>::get().unwrap().block_height).collect::<Vec<_>>().len() as u32;
+			let dot_channels: u32 = pallet_cf_ingress_egress::DepositChannelLookup::<Runtime, PolkadotInstance>::iter().filter(|(_key, elem)| elem.expires_at > pallet_cf_chain_tracking::CurrentChainState::<Runtime, PolkadotInstance>::get().unwrap().block_height).collect::<Vec<_>>().len() as u32;
+			let arb_channels: u32 = pallet_cf_ingress_egress::DepositChannelLookup::<Runtime, ArbitrumInstance>::iter().filter(|(_key, elem)| elem.expires_at > pallet_cf_chain_tracking::CurrentChainState::<Runtime, ArbitrumInstance>::get().unwrap().block_height).collect::<Vec<_>>().len() as u32;
+
+			OpenDepositChannels{
+				ethereum: eth_channels,
+				bitcoin: btc_channels,
+				polkadot: dot_channels,
+				arbitrum: arb_channels,
+			}
+		}
+		fn cf_fee_imbalance() -> FeeImbalance {
+			let eth = pallet_cf_ingress_egress::WithheldTransactionFees::<Runtime, EthereumInstance>::iter().fold(0, |acc, elem| acc+ elem.1) -
+				pallet_cf_broadcast::TransactionFeeDeficit::<Runtime, EthereumInstance>::iter().fold(0, |acc, elem| acc + elem.1);
+			let dot = pallet_cf_ingress_egress::WithheldTransactionFees::<Runtime, PolkadotInstance>::iter().fold(0, |acc, elem| acc+ elem.1) -
+				pallet_cf_broadcast::TransactionFeeDeficit::<Runtime, PolkadotInstance>::iter().fold(0, |acc, elem| acc + elem.1);
+			let btc = pallet_cf_ingress_egress::WithheldTransactionFees::<Runtime, BitcoinInstance>::iter().fold(0, |acc, elem| acc+ elem.1) -
+				pallet_cf_broadcast::TransactionFeeDeficit::<Runtime, BitcoinInstance>::iter().fold(0, |acc, elem| acc + elem.1);
```

Comment:
Does this make sense for bitcoin?

Reply:
Oh yeah forgot for btc doesn't work, removed!